### PR TITLE
Refactor docs to remove reference to file UI encrypting

### PR DESCRIPTION
**Pull request by VeArnold**

### CODE REVIEW

File: content/building/encrypting.md [hunk 1]
```diff
@@ -22,11 +22,17 @@ The encrypted value will look something like this:
 Encrypted(Z0FBQUFBQmRyY1FLWXIwVEhqdWphdjRhQ0xubkdoOGJ2bThkNmh4YmdXbFB3S2wyNTN2OERoV3c0YWU0OVBERG42d3Rfc2N0blNDX3FfblZxbUc4d2pWUHJBSVppbXNXNC04U1VqcGlnajZ2VnJVMVFWc3lZZ289)
 ```
 
-Note that when you upload a **file** for encryption, it is encoded to `base64` and would have to be decoded during the build. If you want to encrypt a file (plaintext, JSON, plist), copy-paste the contents of it to the encryption interface and use the encrypted value in your configuration.
+If you wish to encrypt a **file** to add to your workflow, you will first have to base64 encode it, and then encrypt the received string. To use the file you will have to decode it during the build.
```
Comment (helinanever) [inline content/building/encrypting.md:25]:
```suggestion
If you wish to encrypt a **file** to add to your workflow, you will first have to base64-encode it and then encrypt the received string. To use the file you will have to decode it during the build.
```

Comment (helinanever) [reply content/building/encrypting.md:25]: Is it important to clarify at what stage the decoding should happen in the build? That is where in the yaml they should have the line to decode the file?

File: content/building/encrypting.md [hunk 2]
```diff
@@ -22,11 +22,17 @@ The encrypted value will look something like this:
...
 
-Writing the base64-encoded environment variable to a file can be done like this:
+With MacOS, in order to base64 encode a file and copy the contents to your clipboard, you can run the following command in the terminal:
```
Comment (helinanever) [inline content/building/encrypting.md:27]: With MacOS > On macOS

File: content/building/encrypting.md [hunk 3]
```diff
@@ -22,11 +22,17 @@ The encrypted value will look something like this:
...
 
-```yaml
-scripts:
-  - echo $MY_FILE | base64 --decode > my-file.json
-```
\ No newline at end of file
+  ```bash
+  cat your_file_name.extension | base64 | pbcopy 
+  ```
+After the command has been run, you can paste the string from your clipboard to our encryption tool and then save the encrypted string to your environment variables.
```
Comment (helinanever) [inline content/building/encrypting.md:32]: ... to an environment variable.
```suggestion
After the command has been run, you can paste the string from your clipboard to our encryption tool and then save the encrypted string to an environment variable.
```

File: content/building/encrypting.md (hunk 1 above)
Comment (helinanever) [reply content/building/encrypting.md:25]: Ok, I saw you mention later in the doc that it should be done in scripts.

File: content/building/encrypting.md [hunk 4]
```diff
@@ -22,11 +22,17 @@ The encrypted value will look something like this:
...
+
+Once done, you can base64 decode it during build time in your scripts section using the following command:
```
Comment (helinanever) [inline content/building/encrypting.md:34]:
```suggestion
Finally, base64 decode it during build time in your scripts section using the following command:
```

File: content/code-signing-yaml/signing-ios.md [hunk 5]
```diff
@@ -62,13 +62,11 @@ environment:
...
+  This is the private API key downloaded from App Store Connect. Encrypt the **contents** of the file and save the encrypted value to the environment variable.
 
 - `CERTIFICATE_PRIVATE_KEY`
 
-  A RSA 2048 bit private key to be included in the [signing certificate](https://help.apple.com/xcode/mac/current/#/dev1c7c2c67d) that Codemagic creates. You can use an existing key or create a new 2048 bit RSA key by running the following command in your terminal. 
-  
-  Similarly to the above note that when encrypting files via the UI, they will be base64 encoded and would have to be decoded during the build. Alternatively, you can encrypt the **contents** of the file and save the encrypted value to the environment variable.
+  A RSA 2048 bit private key to be included in the [signing certificate](https://help.apple.com/xcode/mac/current/#/dev1c7c2c67d) that Codemagic creates. You can use an existing key or create a new 2048 bit RSA key by running the command below in your terminal. Similarly to above, note that you should encrypt the **contents** of the file and save the encrypted value to the environment variable.
```
Comment (helinanever) [inline content/code-signing-yaml/signing-ios.md:69]:
```suggestion
  A RSA 2048 bit private key to be included in the [signing certificate](https://help.apple.com/xcode/mac/current/#/dev1c7c2c67d) that Codemagic creates. You can use an existing key or create a new 2048 bit RSA key by running the command below in your terminal. Note that you should encrypt the **contents** of the file and save the encrypted value to the environment variable. 
```

**Comment by helinanever** [comment]
Can you also fix this part?
>Click Encrypt variables on the YAML editor window (or click Encrypt environment variables on the right sidebar in the Configuration as code section if you are using the Flutter workflow editor).

The interface is no longer part of the YAML editor and is on the sidebar for all app types.

**Comment by VeArnold** [comment]
@helinanever Are you sure? looks very attached to the editor for me on live.

**Comment by helinanever** [comment]
@VeArnold you're right, discared that comment.

File: content/building/encrypting.md [hunk 6]
```diff
@@ -22,11 +22,17 @@ The encrypted value will look something like this:
 Encrypted(Z0FBQUFBQmRyY1FLWXIwVEhqdWphdjRhQ0xubkdoOGJ2bThkNmh4YmdXbFB3S2wyNTN2OERoV3c0YWU0OVBERG42d3Rfc2N0blNDX3FfblZxbUc4d2pWUHJBSVppbXNXNC04U1VqcGlnajZ2VnJVMVFWc3lZZ289)
 ```
 
-Note that when you upload a **file** for encryption, it is encoded to `base64` and would have to be decoded during the build. If you want to encrypt a file (plaintext, JSON, plist), copy-paste the contents of it to the encryption interface and use the encrypted value in your configuration.
+If you wish to encrypt a **file** to add to your workflow, you will first have to base64 encode it and then encrypt the received string. To use the file, you will have to decode it during the build.
 
-Writing the base64-encoded environment variable to a file can be done like this:
+On MacOS, in order to base64 encode a file and copy the contents to your clipboard, you can run the following command in the terminal:
```
Comment (helinanever) [inline content/building/encrypting.md:27]:
```suggestion
On macOS, in order to base64 encode a file and copy the contents to your clipboard, you can run the following command in the terminal:
```

**Review by helinanever — APPROVED** [review]
LGTM

**Pull request merged**